src/dmd/backend/cod3.d

```diff
@@ -6784,26 +6785,24 @@ nomatch:
 
 }
 
-/**************************
- * Write code to intermediate file.
- * Code starts at offset.
- * Returns:
- *      addr of end of code
+/************************
+ * Little buffer allocated on the stack to accumulate instruction bytes to
+ * later be sent along to objmod
  */
-
 private struct MiniCodeBuf
 {
 nothrow:
-    size_t index;
-    size_t offset;
+    uint index;
+    uint offset;
```

Comment:
Why change `size_t` to `uint`?

Reply:
reduce stack size consumption

Reply:
A PR should do one thing, not mix adding a feature and doing optimizations

Reply:
2 lines out of 4000 is not a problem.

Reply:
So if this were to introduce a regression and you would git bisect, you would debug a diff of 4000 lines. This is exactly the thing you are vehemently against in other people's PRs.

Reply:
4000 lines? Not at all. This code is not part of the disassembler, I was very careful in isolating the disassembler from internals to the rest of the backend. The changes to the MiniCodeBuf are minimal and easily reviewed.

Secondly, I point out repeatedly that rules for doing things often conflict, and one must always exercise *good judgement*. My complaints were about PRs that would have one line of bug fix and 400 lines of refactorings. Yes, this sort of thing happened repeatedly. If you check my comment history, I don't complain about a line here and there being improved in the context of a larger change.

Thirdly, changing the type got rid of a couple ugly casts.